src/field.rs

```diff
@@ -7,60 +7,45 @@ macro_rules! debug_assert_bits {
     }
 }
 
-#[macro_export]
-macro_rules! field_const_raw {
-    ($d9: expr, $d8: expr, $d7: expr, $d6: expr, $d5: expr, $d4: expr, $d3: expr, $d2: expr,
-     $d1: expr, $d0: expr) => {
-        $crate::curve::Field {
-            n: [$d0, $d1, $d2, $d3, $d4, $d5, $d6, $d7, $d8, $d9],
+#[derive(Debug, Clone)]
+/// Field element for secp256k1.
+pub struct Field {
+    n: [u32; 10],
```

Comment:
independen of this change, it might make sense to be explicit what is the array format LE/BE (not the u32 format, but what is least significant `u32` piece of `n`)